misc/mail.md

```diff
@@ -68,34 +68,40 @@ Computed Properties:
 
 ## SMTP transport events
 
-**Send outbound e-mails**\
-  Trigger: every few minutes if there are any queued outbound messages waiting.
+### Send outbound e-mails
```

Comment:
@martinmacko47 namiesto bold textu + zalomenie riadku som pouzil Heading 3, moze byt?

Reply:
Skor to sprav cez odrazky ako sekciu [Administration v `attachments.md`]. Lebo ked je nadpisov privela, tak sa v tom neda vyznat.

[Administration v `attachments.md`]: https://github.com/martinmacko47/chcemvediet/blob/master/misc/attachments.md#administration

Reply:
:ok: